File: Exercicios/Exerc¡cio_Desplugado/Exercicio.c
/* 1) Escreva um algoritmo que receba 5 números, armazene em um vetor, imprima na tela os valores, ordene esses
valores do maior para o menor e por fim reimprima os valores do vetor ordenados. */

#include <stdio.h>

int compare(const void *a, const void *b) {
    return (*(int*)b - *(int*)a);
}

int main() {
    int N1, N2, N3, N4, N5;
    int lista[5];

    printf("-------------- Algoritmo ----------\n");
    printf("Digite o primeiro numero: ");
    scanf("%d", &N1);
    printf("Digite o segundo numero: ");
    scanf("%d", &N2);
    printf("Digite o terceiro numero: ");
    scanf("%d", &N3);
    printf("Digite o quarto numero: ");
    scanf("%d", &N4);
    printf("Digite o quinto numero: ");
    scanf("%d", &N5);

    lista[0] = N1;
    lista[1] = N2;
    lista[2] = N3;
    lista[3] = N4;
    lista[4] = N5;

    // Ordenando o vetor em ordem decrescente
    qsort(lista, 5, sizeof(int), compare);

    printf("Vetor ordenado do maior para o menor: ");
    for (int i = 0; i < 5; i++) {
        printf("%d ", lista[i]);
    }
    printf("\n");

}

File: Exercicios/Exercicio_Ordenado/Exercicio.py
# 1) Escreva um algoritmo que receba 5 números, armazene em um vetor, imprima na tela os valores, ordene esses
# valores do maior para o menor e por fim reimprima os valores do vetor ordenados.

print("-------------- Algoritmo ----------")
N1 = int(input("Digite o primeiro numero: "))
N2 = int(input("Digite o segundo numero: "))
N3 = int(input("Digite o terceiro numero: "))
N4 = int(input("Digite o quarto numero: "))
N5 = int(input("Digite o quinto numero: "))

lista = [N1, N2, N3, N4, N5]
print("Vetor inicial:", lista)
lista.sort(reverse=True) 
print("Vetor ordenado:", lista)

